.vscode/.cortex-debug.peripherals.state.json

```diff
@@ -0,0 +1 @@
+[{"node":"CAN0.CREL","expanded":true,"format":0},{"node":"SERCOM4.SPIS","expanded":true,"format":0}]
```

Comment:
Should this file be gitignored?

Reply:
Yes it should be, I'll fix that. Also, I'm going to add another launch config to build and load the release.